common/nodes.js

```diff
@@ -265,6 +265,13 @@ module.exports = [
     port: 11625,
     publicKey: "GCWJKM4EGTGJUVSWUJDPCQEOEP5LHSOFKSA4HALBTOO4T4H3HCHOM6UX"
   },
+  {
+    id: "smartlands",
+    name: "Smartlands Platform",
+    host: "stellar.smartlands.io",
+    port: 11625,
+    publicKey: "GCBAQCWXKIIOQOEDIW6EB7GY6UREHPTKK26NZFLVK63ZTRFSXDVX2AHV"
+  },
```

Comment:
Add this as the last entry in this file, with the `,` appended.

Reply:
@bartekn added this as the last entry in this file, with the , appended.